Automattic/sensei
# Add landing page pattern with image

Implements https://github.com/Automattic/sensei/issues/6190

### Changes proposed in this Pull Request

* Add a landing page with image in the header

### Testing instructions

- Have the latest trunk of the [Course theme](https://github.com/Automattic/course)
- Create a new page
- Open the block inserter, then go to patterns and select Sensei LMS from the dropdown
- Select the Landing Page with Image pattern
- Select the `Wide - No Title, No Featured Image` template
- Save and make sure the frontend matches the design

### Screenshot / Video

![Screen Shot 2022-11-29 at 10 44 38 PM](https://user-images.githubusercontent.com/6820724/204591197-4693b3d1-e977-4f33-9599-e58b4b310e9d.png)
